libass/ass_drawing.c

```diff
@@ -265,29 +265,17 @@ void ass_drawing_set_text(ASS_Drawing *drawing, char *str, size_t len)
 }
 
 /*
- * \brief Create a hashcode for the drawing
- * XXX: To avoid collisions a better hash algorithm might be useful.
- */
-void ass_drawing_hash(ASS_Drawing *drawing)
-{
-    if (!drawing->text)
-        return;
-    drawing->hash = fnv_32a_str(drawing->text, FNV1_32A_INIT);
-}
```

Comment:
If the hash is unused, how is it cached?

Reply:
Cache backend is already hashing `text`, such secondary hashing is redundant. Looks like very old artifact for me.